js/wazirx.js

```diff
@@ -0,0 +1,709 @@
+            request['symbol'] = market['id'];
+        }
+        const response = await this.spotV1PrivateGetOpenOrders (this.extend (request, params));
+        // [
+        //     {
+        //         "id": 28,
+        //         "symbol": "wrxinr",
+        //         "price": "9293.0",
+        //         "origQty": "10.0",
+        //         "executedQty": "8.2",
+        //         "status": "cancel",
+        //         "type": "limit",
+        //         "side": "sell",
+        //         "createdTime": 1499827319559,
+        //         "updatedTime": 1499827319559
+        //     },
+        //     {
+        //         "id": 30,
+        //         "symbol": "wrxinr",
+        //         "price": "9293.0",
+        //         "stopPrice": "9200.0",
+        //         "origQty": "10.0",
+        //         "executedQty": "0.0",
+        //         "status": "cancel",
+        //         "type": "stop_limit",
+        //         "side": "sell",
+        //         "createdTime": 1499827319559,
+        //         "updatedTime": 1507725176595
+        //     }
+        // ]
+        const orders = this.parseOrders (response, market, since, limit);
+        return orders;
+    }
+
+    async cancelAllOrders (symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelAllOrders requires a `symbol` argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        return await this.spotV1PrivateDeleteOpenOrders (this.extend (request, params));
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder requires a `symbol` argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'orderId': id,
+        };
+        const response = await this.spotV1PrivateDeleteOrder (this.extend (request, params));
+        return this.parseOrder (response);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'side': side,
+            'type': type,
+            'quantity': amount,
+        };
+        if ((type === 'limit') || (type === 'stop_limit')) {
+            if (price === undefined) {
+                throw new ArgumentsRequired (this.id + ' createOrder requires a price argument for ' + type + ' orders');
+            }
+            request['price'] = this.priceToPrecision (symbol, price);
+            if (type === 'stop_limit') {
+                const stopPrice = this.safeString2 (params, 'stopPx', 'stopPrice');
```

Comment:
If `stopPrice` is defined, then it's a stop-order. If the `price` argument is also defined, then it's a stop-limit order. If `price` is not defined, then it's a stop-market order:

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L3308

Reply:
@kroitor I'll adjust it accordingly for the limit and stop-limit scenarios but if the stopPrice is defined but the price doesn't won't matter because only limit and stop-limit orders are supported.